jetson/nucleo_bridge/LCMHandler.cpp

```diff
@@ -20,7 +20,7 @@ void LCMHandler::init()
     // Subscription to lcm channels
     lcm_bus->subscribe("/ra_ik_cmd", &LCMHandler::InternalHandler::ra_closed_loop_cmd, internal_object);
     lcm_bus->subscribe("/sa_ik_cmd", &LCMHandler::InternalHandler::sa_closed_loop_cmd, internal_object);
-    lcm_bus->subscribe("/carousel_openloop_cmd", &LCMHandler::InternalHandler::carousel_closed_loop_cmd, internal_object);
```

Comment:
So nucleo bridge is only gonna handle open loopo?

Reply:
nucleo bridge should handle closed loop....i think this was a bug where he subscribed to openloop but called the closed loop callback

Reply:
lol this is why travis-free is failing travis-CI, good fix

Reply:
I'll add closed loop soon, still thinking about how we want to do it (prob with limit switches again I guess)

Reply:
probs just the encoder, it's already on the carousel motor, you'll just need it's CPR and it'll need a start up calibration &/or a limit switch to zero it